docs/cookbook-recipes/isle-v140-git-cleanup.md

```diff
@@ -78,7 +78,8 @@ Enter the name here:
 * The script will continue and the process takes about 30 - 45 seconds depending on the speed of your system with minimal output.
 
 * Your final step is to push this local smaller repo back to the remote repository
-* `git push origin master`
+  * `cd ISLE.git` or `cd yourprojectnamehere-isle.git`
+  * `git push origin master`
 
 * Clone down your newly changed and smaller forked or cloned ISLE project directory again to ensure that the repositiory is smaller to a location of your choice.
 
```

Comment:
Can you make the next line bold?  At least the "Remove all previous versions of your cloned ISLE project directory/directories."  And I think you can take out 'larger size' and 'forked', as I think they will all be clones and 'larger size' and 'previous' are redundant.

Reply:
And below that, I think it should read: "Work from a newly cloned ISLE/yourprojecthere-ISLE project directory".

Reply:
Changed as agreed. Over to you for resolution and approval.